draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1827,6 +1827,14 @@ that are mandatory to implement for CT-using TLS clients (see
 if it did not include both the `transparency_info` and `status_request` TLS
 extensions in the ClientHello.
 
+## TLS Server {#no_tls_servers}
+
+This specification includes no description of what a TLS server should
+do if it wants to verify a received TLS client certificate.
```

Comment:
I presume that "if it wants to verify" is intended to hint at the idea of using CT for TLS client certificates, but perhaps this could be stated more explicitly?
It might be worth pointing out that, in general, TLS client certificates are not expected to be submitted to CT logs, since "In this document, we only describe its use for public TLS server certificates" (see the Introduction).